MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/query/BuildingOutlineActivity.java

```diff
@@ -128,13 +128,15 @@ private LineString getBuildingFeatureOutline(@NonNull Style style) {
       if (features.size() > 0) {
         Feature buildingFeature = features.get(0);
         // Build a list of Point objects from the building Feature's coordinates
-        for (int i = 0; i < ((Polygon) buildingFeature.geometry()).coordinates().size(); i++) {
-          for (int j = 0;
-               j < ((Polygon) buildingFeature.geometry()).coordinates().get(i).size(); j++) {
-            pointList.add(Point.fromLngLat(
-              ((Polygon) buildingFeature.geometry()).coordinates().get(i).get(j).longitude(),
-              ((Polygon) buildingFeature.geometry()).coordinates().get(i).get(j).latitude()
-            ));
+        if (buildingFeature.geometry() instanceof Polygon) {
+          for (int i = 0; i < ((Polygon) buildingFeature.geometry()).coordinates().size(); i++) {
```

Comment:
NIT: you can assign `buildingFeature` to a local `Polygon` type variable to clean up all of those casts.